buildSrc/settings.gradle.kts

```diff
@@ -74,12 +74,12 @@ gradle.settingsEvaluated {
         return@settingsEvaluated
     }
 
-    if (remoteBuildCacheEnabled(this) && !getBuildJavaVendor().contains("Oracle") && !JavaVersion.current().isJava9()) {
-        throw GradleException("Remote cache is enabled, which requires Oracle JDK 9 to perform this build. It's currently ${getBuildJavaVendor()} at ${getBuildJavaHome()}.")
+    if (remoteBuildCacheEnabled(this) && !(isOpenJDK() && JavaVersion.current().isJava11)) {
```

Comment:
I personally think the condition would be easier to read in the format it was before:
`remoteBuildCacheEnabled(this) && !isOpenJDK() && !JavaVersion.current().isJava11`

Reply:
Yeah... I've created a new method `isOpenJDK11()` to improve readability, thanks.